app/components/shared/searchable_collection_component.rb

```diff
@@ -11,9 +11,18 @@ def initialize(form:, attribute_name:, collection:, value_method:, text_method:,
     @hint_method = hint_method
 
     @small = searchable
+    @scrollable = scrollable || searchable
   end
 
   def searchable
     collection.count >= threshold
   end
+
+  def scrollable_class(scrollable)
```

Comment:
you don't need to pass an argument in because the method will already have access to them (same below too)